certified-connectors/Tilkee-V3/apiProperties.json

```diff
@@ -0,0 +1,36 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "ToBeGeneratedOnTilkee",
```

Comment:
@FloxoS Please update it to `"[[TEST]]` to maintain the consistency with other connectors.